### PR TITLE
Added new icons and added flat button

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Changes in this PR:

1. Updating all the latest icons
Screenshot 2020-09-24 at 10 12 08 PM
Screenshot 2020-09-24 at 10 12 25 PM
Screenshot 2020-09-24 at 10 12 37 PM

2. Adding flat buttons
Screenshot 2020-09-24 at 10 13 00 PM

3. Updating link button CSS with style guide
Screenshot 2020-09-24 at 10 14 37 PM

4. Added new color - dustygray
Screenshot 2020-09-24 at 10 20 28 PM

5. Other minor changes in buttons, modal and form

### CODE REVIEW

**Review by is-ashish — APPROVED** [review]
LGTM

**Review by avishnu — APPROVED** [review]
Lgtm

File: scss/core/abstracts/_variables.scss [hunk 1]
```diff
@@ -75,9 +75,9 @@ $body-color:                $gray-600 !default;
 //
 // Style anchor elements.
 
-$link-color:                theme-color("primary") !default;
+$link-color:                $blue-500 !default;
 $link-decoration:           none !default;
-$link-hover-color:          darken($link-color, 15%) !default;
+$link-hover-color:          $blue-500 !default;
```
Comment (IsAmrish) [inline scss/core/abstracts/_variables.scss:80]: @Pallavi-PH  -- isn't it a good idea to change the color a little bit on hover over the link?

**Review by IsAmrish — APPROVED** [review]
LGTM

**Pull request merged**